moped-editor/src/layouts/DashboardLayout/TopBar.js

```diff
@@ -19,6 +19,30 @@ import DropdownMenu from "./NavBar/DropdownMenu";
 import NavigationSearchInput from "./NavBar/NavigationSearchInput";
 import NavLink from "src/components/NavLink";
 
+const getAlertBannerSeverity = (env) => {
+  // show an orange banner on local
+  // show blue on staging, netlify, test, ...not production
+  switch (env) {
+    case "local":
+      return "warning";
+    default:
+      return "info";
+  }
+};
+
+const EnvAlertBanner = () => {
+  const env = process.env.REACT_APP_HASURA_ENV;
+  if (env === "production") {
+    return null;
+  }
+  return (
+    <Alert severity={getAlertBannerSeverity(env)}>
+      This is a <span style={{ fontWeight: "bold" }}>{env}</span> environment
+      for testing purposes.
+    </Alert>
+  );
+};
+
```

Comment:
should i move this component to a separate file?

Reply:
I'm fine either way!

Reply:
I think that it is good to stay since it isn't being used anywhere else 👍